base/strings/string.jl

```diff
@@ -192,7 +199,8 @@ end
     ((i += 1) > n) | (l < 0xf0) && return i
     # third continuation byte
     @inbounds b = codeunit(s, i)
-    ifelse(b & 0xc0 ≠ 0x80, i, i+1)
+    return ifelse(b & 0xc0 ≠ 0x80, i, i+1)
+    end)(s, i, n, l)
```

Comment:
I didn't really want it to be seen as a separate function, since it is just an outlined block of code code but still sequentially part of the same logical flow

Reply:
I agree with @ararslan that the current way of writing it looks very confusing though, especially when the call is so far away from the `function` definition and there is no indentation.